htlcswitch/hop/iterator_test.go

```diff
@@ -65,14 +65,16 @@ func TestSphinxHopIteratorForwardingInstructions(t *testing.T) {
 			},
 			expectedFwdInfo: expectedFwdInfo,
 		},
-		// A TLV payload, we can leave off the action as we'll always
-		// read the cid encoded.
+		// A TLV payload, which includes the sphinx action as
+		// cid may be zero for blinded routes (thus we require the
+		// action to signal that we are at the final hop).
```

Comment:
Referring to the general mechanism of using the action rather than the specifics of this test case. 

Would "signal that we are" -> "signal whether we are" make it clearer?